app/components/CoinListCell/CoinListCellFavoritesButton/CoinListCellFavoritesButton.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+import { View, Text, TouchableOpacity, Image } from 'react-native'
+import coinListCellFavoritesButtonStyles from './CoinListCellFavoritesButtonStyles'
+
+export default class CoinListCellFavoritesButton extends Component {
+state ={
```

Comment:
Need this?

Reply:
yeah I left it in here because we're going to need to trigger a re-render when the button gets tapped and the styling will change to reflect that.